extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -139,7 +139,7 @@ export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
                     let logger = _self.diagLog();
                     _contentHandler = new DomContentHandler(_config, logger);
                     let metaTags = _contentHandler.getMetadata();
-                    _pageAction = new PageAction(this, _config, _contentHandler, _config.callback.pageActionPageTags, metaTags, logger);
+                    _pageAction = new PageAction(_self, _config, _contentHandler, _config.callback.pageActionPageTags, metaTags, logger);
```

Comment:
This is the source of the issue in that when the `onConfigChange` is re-called the `this` is undefined which then causes the creation of a new `PageAction` with `undefined` as the ClickAnalytics instance.

I've also looked for a "fixed" other potential issues of the same in the other classes. This doesn't replace "all" of the `this` usages as there are some where the `this` does need to represent the current execution context (ie. `this`)